data/afaanoromoo/articles/c4g19kgl85ko.json

```diff
@@ -7,22 +7,29 @@
     "type": "article",
     "createdBy": "Afaanoromoo",
     "language": "om",
-    "lastUpdated": 1562936158365,
+    "lastUpdated": 1570031976502,
     "firstPublished": 1562936153914,
-    "lastPublished": 1562936153914,
+    "lastPublished": 1569857685157,
     "options": {},
     "analyticsLabels": {},
     "passport": {
       "language": "om",
       "home": "http://www.bbc.co.uk/ontologies/passport/home/AfaanOromoo"
     },
     "tags": {},
-    "version": "v1.1.0",
+    "version": "v1.1.2",
```

Comment:
Was the jump from `v1.1.0` to `v1.1.2` in all of these files intentional?

Reply:
@simonsinclair I believe this is the Ares response version. It has updated since the last time we looked at the Ares Broker Test environment.